src/Hedgehog/Property.fs

```diff
@@ -145,32 +172,29 @@ module Property =
                 | Failure ->
                     { Tests = tests + 1<tests>
                       Discards = discards
-                      Status = takeSmallest renderRecheck size seed result 0<shrinks> }
+                      Status = takeSmallest renderRecheck size seed result 0<shrinks> config.ShrinkLimit}
                 | Success () ->
                     loop seed2 (nextSize size) (tests + 1<tests>) discards
                 | Discard ->
                     loop seed2 (nextSize size) tests (discards + 1<discards>)
 
         loop seed size0 0<tests> 0<discards>
 
-    let private reportWith (renderRecheck : bool) (size : Size) (seed : Seed) (p : Property<unit>) : Report =
-        reportWith' renderRecheck size seed 100<tests> p
-
-    let report' (n : int<tests>) (p : Property<unit>) : Report =
+    let reportWith (config : PropertyConfig) (p : Property<unit>) : Report =
         let seed = Seed.random ()
-        reportWith' true 1 seed n p
+        reportWith' true 1 seed config p
```

Comment:
We should consider inlining `reportWith'` here. This isn't a requirement for this PR, however.

Reply:
Yeah let's leave that to another PR. I'm trying to keep this small (and failing).